src/features/payment-agreements/hooks/mutations/use-create-payment-agreement-mutation.ts

```diff
@@ -27,6 +27,8 @@ export function useCreatePaymentAgreementMutation() {
     onSuccess(sessionId) {
       if(!sessionId) return Promise.reject();
 
+      client.resetQueries({queryKey: accountPaymentAgreementsCreatedQueryKey(address), exact: true})
```

Comment:
This should be added in the getCallback function, along with the other query invalidations